variables.tf

```diff
@@ -16,7 +16,8 @@
 
 variable "random_project_id" {
   description = "Enables project random id generation. Mutually exclusive with project_id being non-empty."
-  default     = "false"
+  type = bool
```

Comment:
formatting

Reply:
Not related to your change, please revert

Reply:
Reverted